gefserver/pier/internal/dckr/dckr.go

```diff
@@ -395,7 +402,9 @@ func (c Client) StartImage(id string, repoTag string, cmdArgs []string, binds []
 	<-attached
 	attached <- struct{}{}
 
-	err = c.c.StartContainer(cont.ID, &hc)
+	jobExecutionContext, cancel := context.WithTimeout(context.Background(), time.Duration(executionTimeOut)*time.Second)
+	defer cancel()
```

Comment:
StartContainer cannot accept a context, StartContainerWithContext is the same thing but with a context